aip/orderby_test.go

```diff
@@ -0,0 +1,78 @@
+package aip
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	testUtils "github.com/CDNA-Technologies/go-utils/testing"
+	pb "github.com/CDNA-Technologies/proto-gen/go/gonuclei/masterdata/v2"
+	"go.einride.tech/aip/ordering"
+)
+
+func TestParseAndValidateOrderBy(t *testing.T) {
+	tests := []struct {
+		req     ordering.Request
+		want    map[string]string
+		wantErr error
+	}{
+		{
+			req: &pb.ListPartnersRequest{
+				OrderBy: "",
+			},
+			want:    make(map[string]string),
+			wantErr: nil,
+		},
+		{
+			req: &pb.ListPartnersRequest{
+				OrderBy: "?",
+			},
+			want:    nil,
+			wantErr: fmt.Errorf("invalid orderby : ?"),
+		},
+		{
+			req: &pb.ListPartnersRequest{
+				OrderBy: "name desc age asc",
+			},
+			want:    nil,
+			wantErr: fmt.Errorf("invalid orderby : name desc age asc"),
+		},
+		{
+			req: &pb.ListPartnersRequest{
+				OrderBy: "name desc",
+			},
+			want: map[string]string{
+				"name": "DESC",
+			},
+			wantErr: nil,
+		},
+		{
+			req: &pb.ListPartnersRequest{
+				OrderBy: "name desc, age",
+			},
+			want: map[string]string{
+				"name": "DESC",
+				"age":  "",
+			},
+			wantErr: nil,
+		},
+		{
+			req: &pb.ListPartnersRequest{
+				OrderBy: "name Desc, age",
+			},
+			want:    nil,
+			wantErr: fmt.Errorf("invalid orderby : name Desc, age"),
+		},
+	}
+	for _, tt := range tests {
```

Comment:
```suggestion
	for _, input := range tests {
```
Let's keep it consistent

Reply:
Fixed for all test cases